server/migrations/20171226132000-create-todo.js

```diff
@@ -0,0 +1,33 @@
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('Todos', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER,
+      },
+      title: {
```

Comment:
I want you to make the `title` unique. You should check the docs for how to do that.